.github/workflows/project-high-level-tracker.yaml

```diff
@@ -0,0 +1,268 @@
+name: RAPIDS High Level Tracking
+
+on:
+  workflow_call:
+    inputs:
+      PROJECT_ID:
+        description: "Project ID"
+        default: "PVT_kwDOAp2shc4ArK9w"
+        type: string
+      
+      # ------------- Field IDs -------------
+      WEEK_FIELD_ID:
+        description: "Week Field ID"
+        default: "PVTIF_lADOAp2shc4ArK9wzgiTHg8"
+        type: string
+      RELEASE_FIELD_ID:
+        description: "Release Field ID"
+        default: "PVTF_lADOAp2shc4ArK9wzgpEYKw"
+        type: string
+      STAGE_FIELD_ID:
+        description: "Stage Field ID"
+        default: "PVTSSF_lADOAp2shc4ArK9wzgiTH8g"
+        type: string
+      START_DATE_FIELD_ID:
+        description: "Start Date Field ID"
+        default: "PVTF_lADOAp2shc4ArK9wzgiTHmQ"
+        type: string
+      END_DATE_FIELD_ID:
+        description: "End Date Field ID"
+        default: "PVTF_lADOAp2shc4ArK9wzgiTHmU"
+        type: string
+
+      # ------------- Option IDs -------------
+      # Stage options
+      STAGE_PLANNED_ID:
```

Comment:
I'm a little confused by the naming with `*_ID` for these things... they look like names or labels. I think changing the input names to something like `*_NAME` would make it clearer that these are free-text display names that don't have uniqueness constraints.

Am I right about that? If so, would you consider that naming change?

Reply:
Yeah we can probably change these; we use the field name as the ID in the query (since they added the capability to do so semi-recently).

Making a note here to change it so it's clearer.

Reply:
Updated in https://github.com/rapidsai/shared-workflows/pull/288/commits/1d7d605961b1d5a0a418752e08953f935d1fb28f